react/components/SideBarItem.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, FunctionComponent } from 'react'
+
+import { Link } from 'vtex.render-runtime'
+import { IconCaretDown, IconCaretUp } from 'vtex.styleguide'
+
+interface Props {
+  text: string
+  link?: string
+  appName: string
+  hasArticles?: boolean
+}
+
+const SideBarItem: FunctionComponent<Props> = ({
+  text,
+  link,
+  appName,
+  hasArticles,
+  children,
+}) => {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <li className="link">
+      <div className="flex justify-between items-center">
+        {link ? (
+          <Link to={`/docs/${appName}/${link}`}>
+            <p>{text}</p>
+          </Link>
+        ) : (
+          <p>{text}</p>
+        )}
+        <div
+          className="ph4"
+          onClick={() => setOpen(!open)}
+          onKeyPress={() => setOpen(!open)}
+          role="menuitem"
+          tabIndex={0}>
```

Comment:
Use -1 instead, so it will follow tabIndex natural order

Reply:
Oooh! Did not know that hahahaha